src/Tests.Microsoft.Security.Utilities.Core/RegexPatternTests.cs

```diff
@@ -425,4 +425,25 @@ public void RegexPatterns_RaisesArgumentException_OnNullPattern()
             new RegexPattern(Id, Name, DetectionMetadata.HighEntropy, null);
         });
     }
+
+    [TestMethod]
+    public void RegexPattern_RegexOptions_UsesDefault()
+    {
+        var pattern = new RegexPattern("id", "name", DetectionMetadata.None, ".");
+        pattern.RegexOptions.Should().Be(RegexDefaults.DefaultOptions,
+                                         because: "No regex options were passed so default opptions should be used.");
```

Comment:
```suggestion
                                         because: "No regex options were passed so default opptions should be used.");
```
```suggestion
                                         because: "No regex options were passed so default options should be used.");
```